qutebrowser/config/config.py

```diff
@@ -640,23 +640,32 @@ class StyleSheetObserver(QObject):
     Attributes:
         _obj: The object to observe.
         _stylesheet: The stylesheet template to use.
+        _options: The config options that the stylesheet uses. When it's not
+                  necessary to listen for config changes, this attribute may be
+                  missing.
```

Comment:
An attribute which is only maybe there is weird - just set it to `None` instead.

Reply:
No opinion on this, but the assignment of None would be redundant because it's never checked for.

Reply:
IMHO it's good practice for an object to either have an attribute, or not have it - but not to sometimes have it and sometimes not.